src/domains/sign/cosmos_issuer.tf

```diff
@@ -0,0 +1,38 @@
+module "cosmosdb_sql_database_issuer" {
+  source              = "git::https://github.com/pagopa/azurerm.git//cosmosdb_sql_database?ref=v2.13.1"
+  name                = "issuer"
+  resource_group_name = azurerm_resource_group.data_rg.name
+  account_name        = module.cosmosdb_account.name
+  throughput          = var.io_sign_database_issuer.throughput
+}
+
+module "cosmosdb_sql_container_issuer-dossiers" {
+  source              = "git::https://github.com/pagopa/azurerm.git//cosmosdb_sql_container?ref=v2.13.1"
+  name                = "dossiers"
+  resource_group_name = azurerm_resource_group.data_rg.name
+  account_name        = module.cosmosdb_account.name
+  database_name       = module.cosmosdb_sql_database_issuer.name
+  partition_key_path  = "/subscriptionId"
+  throughput          = var.io_sign_database_issuer.dossiers.throughput
+}
+
+module "cosmosdb_sql_container_issuer-signature-requests" {
+  source              = "git::https://github.com/pagopa/azurerm.git//cosmosdb_sql_container?ref=v2.13.1"
+  name                = "signature-requests"
+  resource_group_name = azurerm_resource_group.data_rg.name
+  account_name        = module.cosmosdb_account.name
+  database_name       = module.cosmosdb_sql_database_issuer.name
+  partition_key_path  = "/subscriptionId"
+  throughput          = var.io_sign_database_issuer.signature_requests.throughput
+}
+
+module "cosmosdb_sql_container_issuer-upload-documents" {
+  source              = "git::https://github.com/pagopa/azurerm.git//cosmosdb_sql_container?ref=v2.13.1"
+  name                = "upload-documents"
```

Comment:
we updated this container name, from `upload-documents` to `uploads`

Reply:
@lucacavallaro is there a document in which I can find these decisions/parameters?